### DISCUSSION TITLE
Discussions vs Issues - Please read before opening a new discussion

### DISCUSSION THREAD

**Discussion by paula-w** [YetiForceCompany/YetiForceCRM#15377] [Announcement]
Hello everyone!

We recently noticed an increasing number of errors related to YetiForce CRM reported in Discussions. Keep in mind that **Discussions are not meant to be used for reporting issues** you are experiencing with the system. 

Please use the **[Issues](https://github.com/YetiForceCompany/YetiForceCRM/issues/new/choose)** feature for reporting all errors you encounter in the system and make sure to fill the Issue template to provide us with all the necessary information we need to help you. 

**Errors reported via the **Discussion** feature WILL NOT be investigated and the Discussion in question will be deleted.**

**Comment by garryatbrighton** [comment]
It is OK to discuss potential error before reporting. Discussions should be a community focus and issues developer.

**Reply by mariuszkrzaczkowski** [reply]
Sure it's OK to discuss, however, very often these issues are related to wrong configuration, version, or environment, and the authors don't often include these details in their discussions, which wastes everybody's time if we need to keep asking for more information.
This is why we added templates to Issues, so that people know what information is necessary for us to better understand the problem. There is a way to convert a Discussion to an Issue while keeping all its contents but the necessary information must be complete - without that there is a chance nobody is going to take care of the issue (including community members) because nobody knows what the problem could possibly be. 
The quickest way to receive the best answer is to ask a specific and complete question.
We gotta help each other so we don't report the same issue multiple times, just described in a slightly different way.
